pydarn/plotting/rtp.py

```diff
@@ -1,6 +1,6 @@
 # Copyright (C) 2019 SuperDARN
 # Author: Marian Schmidt
-# This code is improvement based on rti.py in the DaVitpy library
+# This code is improvement based on rtp.py in the DaVitpy library
```

Comment:
This was correct before, it is based on rti.py davitpy, located at https://github.com/vtsuperdarn/davitpy/tree/master/davitpy/pydarn/plotting

Reply:
Whoops replace and find got me there.